test/map.spec.js

```diff
@@ -42,7 +57,11 @@ test('test fillGridWithSubgrid', t => {
   ];
   map.fillGridWithSubgrid(grid, subgrid, 1, 0);
   t.same(grid, expected);
-
+  t.doesNotThrow(() => { map.fillGridWithSubgrid(grid, subgrid, 0, 0); });
```

Comment:
I think we should do the same for all of these too. Although we might need to talk on slack about this decision. I think test names should be descriptive and specific so that failures are easier to track down